datafusion/expr/src/function.rs

```diff
@@ -37,10 +38,36 @@ pub type ScalarFunctionImplementation =
 pub type ReturnTypeFunction =
     Arc<dyn Fn(&[DataType]) -> Result<Arc<DataType>> + Send + Sync>;
 
-/// Factory that returns an accumulator for the given aggregate, given
-/// its return datatype.
+/// Arguments passed to create an accumulator
+pub struct AccumulatorArgs<'a> {
+    // default arguments
+    pub data_type: &'a DataType, // the return type of the function
+    pub schema: &'a Schema,      // the schema of the input arguments
+    pub ignore_nulls: bool,      // whether to ignore nulls
```

Comment:
Since these field are `pub` I think they show up in the rustdocs. Can you please make them with three `///` and above the field in question to show up in the docs?